libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Queue/CSFNetwork.m

```diff
@@ -197,15 +197,19 @@ - (CSFAction*)duplicateActionInFlight:(CSFAction*)action {
     CSFAction *result = nil;
     
     for (CSFAction *operation in self.queue.operations) {
-        if (![operation isKindOfClass:[CSFAction class]])
+        if (![operation isKindOfClass:[CSFAction class]]) {
             continue;
-        
+        }
+        if ([action.method isEqualToString:@"POST"] || [action.method isEqualToString:@"PUT"]) {
```

Comment:
Disabling duplicate detection logic for ```POST``` and ```PUT```. I fixed a bug around this a while ago, but @khawkins and I discussed this and agreed it doesn't make much sense.